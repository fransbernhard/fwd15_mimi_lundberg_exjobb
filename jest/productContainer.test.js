import React, { Component } from 'react';
import renderer from 'react-test-renderer';
import ProductContainer from '../src/archive';

describe('components', () => {
  describe('<ProductContainer/>', () => {
    it('renders correctly', () => {
      const tree = renderer.create(
        <ProductContainer
          products={[
            {id: 1, category: 'paint', name: 'clowd', type: 'matt emulsion', stocked: true, size: '100x130', thumbnail: '23-sm.png', previewImg: "23.png"},
            {id: 2, category: 'paint', name: 'dÃ¥lig sikt', type: 'matt emulsion/olja/akryl', stocked: true, size: '100x130', thumbnail: '24-sm.png', previewImg: "24.png"}
          ]}
        />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
