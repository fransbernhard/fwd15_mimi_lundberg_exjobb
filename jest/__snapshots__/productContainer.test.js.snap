// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <ProductContainer/> renders correctly 1`] = `
<div>
  <div
    className="prodContainer"
  >
    <div
      className="product hvr-sink"
      onClick={[Function]}
    >
      <noscript />
      <h3>
        clowd
      </h3>
      <p>
        matt emulsion
      </p>
      <p>
        100x130
         cm
      </p>
    </div>
    <div
      className="product hvr-sink"
      onClick={[Function]}
    >
      <noscript />
      <h3>
        d√•lig sikt
      </h3>
      <p>
        matt emulsion/olja/akryl
      </p>
      <p>
        100x130
         cm
      </p>
    </div>
  </div>
</div>
`;
